client/wfprev-war/src/main/angular/src/app/services/map-config.service/map.config.ts

```diff
@@ -25,8 +32,34 @@ export const mapConfig = () => ({
       type: 'search',
       enabled: false,
     },
-
-  ]
+    {
+      type: 'location',
+      enabled: false
+    },
+    {
+      type: 'identify',

```

Comment:
I would suggest to turn identify on, otherwise is diffcult for client to understand the wildfire dots on the map...

Reply:
done